lib/server.js

```diff
@@ -38,6 +34,18 @@ var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
 
 exports.create = function create(createOptions) {
 
+  // safely require the peer-dependencies
+  var React = util.safeRequire('react');
+  var Router = util.safeRequire('react-router');
+
+  if (!React) {
+    throw new Error('React is required as a peer dependency. Run `npm instal react`.')
+  }
+
+  if (!Router) {
+    throw new Error('React Router is required as a peer dependency. Run `npm instal react-router`.')
```

Comment:
I see. Are you saying you want to make react-router optional if no router relevant options are passed?
My concern is if we make react-router optional, is there any other places we need to change?